lib/hipchat/room.rb

```diff
@@ -30,6 +30,8 @@ def get_room
         raise UnknownRoom,  "Unknown room: `#{room_id}'"
       when 401
         raise Unauthorized, "Access denied to room `#{room_id}'"
+      when 403
```

Comment:
Yep. Whenever I hit the rate limit I get a 403. [The documentation](https://www.hipchat.com/docs/api/rate_limiting) also says you will get a 403. Is 429 even a valid HTTP status code?